test/unit/utils.test.ts

```diff
@@ -0,0 +1,19 @@
+import { privateKeyToEncodedPublicKey } from "../../src/utils"
+import data from "../data/utilsData"
+describe("Unit Test Cases for utils", () => {
+    describe("privateKeyToEncodedPublicKey Testcases", () => {
```

Comment:
Would recommend to have the space in `describe` and `test` to increase the readability of the code otherwise there would not be any breathing room.

Reply:
Shall I create a preetier config file for it?

Reply:
Yeah It would great to have but try not to put too much constraint in the prettier, Keep it loosen up like spacing, new line , line end and once we complete whole test cases then we can harden the constraints in prettier.

Reply:
got it